samples/js/mocha/build.gradle

```diff
@@ -51,13 +51,15 @@ compileTestKotlin2Js.configure {
 }
 
 //gather all required dependencies and put them into the node_modules folder of this (sub)-project
-task populateNodeModules(type: Copy, dependsOn: compileKotlin2Js) {
-    from compileKotlin2Js.destinationDir
+task populateNodeModules(type: Copy, dependsOn: compileKotlinJs) {
+    from compileKotlinJs.destinationDir
 
     afterEvaluate {
         // in contrast to kotlin examples we use testRuntimeClasspath this way we also pick up runtimeOnly dependencies
         configurations.testRuntimeClasspath.each {
-            from zipTree(it.absolutePath).matching { include '*.js', '*.js.map' }
+            if (it.name.endsWith("jar")) {
```

Comment:
I really hoped that this hack was not required with the new plugin any more. Are you sure this is still required? Would you mind to try out if it works also without populateNodeModules?

Reply:
I tried without populateNodeModules and it did not work. No node_modules appear in the build directory, and it fails with the error "Cannot find module 'kotlin'" during the runMocha task. I messed around with other options but wasn't able to get it working. It may be possible once kotlin-gradle-plugin is updated from 1.3.71 as it seems like there are lots of changes there.

Reply:
that's a pity. Maybe the problem stems from the fact that we configure node, might be the new plugin (v. 1.3.71) does this on its own?

Reply:
I was able to get it working by removing this node configuration and making some other small adjustments, however there is one complication. The new plugin also generates a yarn.lock file, and the fact that the project is named "mocha" seems to interfere with this. Unless the project name is changed in settings.gradle to something else, it seems like there is a conflict with the actual mocha library or something that causes the yarn.lock to not be created properly, and the following error: "
Execution failed for task ':kotlinNpmInstall'.
> Cannot find mocha@6.2.2 in yarn.lock
"
I tried changing the project name to "mymocha" just to see and there is no such issue, all tests run successfully.

Reply:
Nice 🎉
then lets rename the project in settings to atrium-with-mocha